CODEOWNERS

```diff
@@ -211,6 +211,7 @@ Jenkinsfile.nightly @cilium/ci-structure
 /pkg/sysctl @cilium/bpf
 /pkg/testutils/ @cilium/ci-structure
 /pkg/tuple @cilium/bpf
+/pkg/wireguard @cilium/wireguard
```

Comment:
We need to document this team at the top of the CODEOWNERS file and create it at https://github.com/isovalent/team-management.